src/frontend/Komponenter/Behandling/Brev/FrittståendeBrev.tsx

```diff
@@ -34,15 +32,12 @@ const BrevKolonner = styled.div`
     display: flex;
 `;
 
-const BrevWrapper = styled.div`
-    margin-left: 4rem;
-`;
-
-interface Props {
-    fagsakId: string;
-}
-
-const ManueltBrev: React.FC<Props> = ({ fagsakId }) => {
+type Props = {
+    oppdaterBrevressurs: (brevRessurs: Ressurs<string>) => void;
+    behandlingId?: string;
+    fagsakId?: string;
+};
+const FrittståendeBrev: React.FC<Props> = (props) => {
```

Comment:
Litt usikker på navngiving her. Burde det hete `FritekstBrev` istedenfor `FrittståendeBrev`? Dette er vel en generell komponent for brev med fritekst, som ikke nødvendigvis er frittstående?

Reply:
Fikset i 2690ebf